mjordan/islandora_workbench
# Draft: Example content naming

## Link to Github issue or other discussion

https://github.com/mjordan/islandora_workbench/issues/537

## What does this PR do?

Add the demo content . I think it may have an issue with collections, and putting the content with its parent object.

## What changes were made?

Adds a config file, a directory full of files including a CSV that references the media files within the directory.

## How to test / verify this PR?

Open isle-dc and run the migration. There are a few steps needed. Please see https://github.com/Islandora-Devops/isle-dc/blob/development/Makefile#L372-L385

## Interested Parties

@mjordan 

---

## Checklist


I'm not sure how much of this applies.
